Makefile

```diff
@@ -74,6 +74,8 @@ clean: ## Cleans dependencies, previous builds and temp files
 	@echo Cleanup finished
 
 post-install:
+	@npx jetify
```

Comment:
Is npx installed by default, or is it something we need to add to the setup docs? Otherwise, we could change this to `./node_modules/.bin/jetify`